src/main/java/seedu/todo/controllers/ListController.java

```diff
@@ -206,31 +221,42 @@ private void displayErrorMessage(String input, boolean listAll, boolean listAllS
      *            String input by the user, to display for error message                                            
      */
     private void setupView(boolean isTask, boolean listAll, boolean isCompleted,
```

Comment:
Isn't this method way too complex and big already, considering it only passes the work onto other methods